gulp/util.js

```diff
@@ -85,13 +85,20 @@ function buildModule(module, opts) {
   var name = module.split('.').pop();
   utils.copyDemoAssets(name, 'src/components/', 'dist/demos/');
 
-  var stream = utils.filesForModule(module)
-      .pipe(filterNonCodeFiles())
-      .pipe(gulpif('*.scss', buildModuleStyles(name)))
-      .pipe(gulpif('*.js', buildModuleJs(name)));
+  var stream = utils.filesForModule(module);
+
   if (module === 'material.core') {
+    stream = stream.pipe(filter(function(x) {
+      // We are not injecting the layout-attributes selectors into the core module css,
+      // otherwise we would have the layout-classes and layout-attributes in there.
+      return !/.*layout-attributes\.scss/g.test(x.path);
```

Comment:
The actual issue was, that the layout attributes were also added to the `modules/core/core.css`, in addition to the class layout attributes. That means that both layout opportunities were added into the core stylesheet.

Filtering in `filesForModule` won't be that elegant, because we only want to filter it for the `material.core` module. Adding this check to `filesForModule` would be really unclear.